app/views/users/show.html.erb

```diff
@@ -0,0 +1,22 @@
+<h1>User profile</h1>
+
+<p>
+  <strong>Email:</strong>
+  <%= @user.email %>
+</p>
+
+<p>
+  <strong>First name:</strong>
+  <%= @user.user_profile.first_name %>
+</p>
+
+<p>
+  <strong>Last name:</strong>
+  <%= @user.user_profile.last_name %>
+</p>
+
+
+<p>
+  <strong>SSH-Key:</strong>
+  <%#= @user.user_profile.ssh_key%>
+</p>
```

Comment:
do we need to apply style guides for this page

Reply:
Good question, I think this one still need styling